TRANSLATION.md

```diff
@@ -96,3 +96,28 @@ Translations are synced between Crowdin and the repository via GitHub Actions.
 - Via a cron schedule, and as needed manually by a collaborator, we download completed translations.
 
 Incoming translations are linted to ensure they come from crowdin, and are also formatted to adhere to our project settings.
+
+## Why We Translate This section and Not this Section
+
+> First what we mean by "section" is a part of the website. It's a collection of pages that are related to each other. For example, the "Learn" section is a collection of pages that are related to learning/discovering Node.js.
+
+Here we only talk about md/mdx file. The translation of components is covered by a JSON dictionary, so not an issue at all.
+
+We have these sections on the website:
+
+- Learn
+- About
+- Download
+- Blog
+
+### Sections that are translated
+
+- **About**: This section is translated because it contains information about the Node.js project, its governance. There are no technical terms that are hard to translate.
+- **Download**: This section is translated because it contains information about how to download Node.js. And it's really important that everyone can understand how to download Node.js.
+
+- **Landing page**: The landing page is translated because it's the first page that people see when they visit the website. It's important that everyone can understand what Node.js is and what it can do.
+
+### Sections that are not translated
+
+- **Blog**: The blog is not translated because it's not useful to translate blog posts. Most of the blog posts are release notes, and it's not useful to translate them.
```

Comment:
Add docs
```suggestion
- **Docs**: The API docs are too expansive, and versioned, to be translated.
- **Blog**: The blog is not translated because it's not useful to translate blog posts. Most of the blog posts are release notes, and it's not useful to translate them.
```